torch/csrc/distributed/rpc/future_message.cpp

```diff
@@ -16,14 +19,29 @@ const Message& FutureMessage::wait() {
   return message_;
 }
 
+const Message& FutureMessage::waitNoThrow() {
```

Comment:
Question, why we would want to ignore the exception message for `FORWARD_AUTOGRAD_REQ`?

Reply:
The intention here isn't to ignore the exception, but rather to pass it through, and allow it to be processed at a higher layer. If this isn't appropriate, I'm happy to change, but was concerned about over-processing at this layer.